README.md

```diff
@@ -28,26 +28,29 @@ This content repository allows easy content management via a headless CMS, using
 
 
 ## Releases
-The current LTS release is: 1.0
-
 [Semantic versioning](https://semver.org/) is in use in this project.
-1. Feature releases
-    - Feature releases are a MAJOR.MINOR combination, eg. `1.2`
-    - Feature releases occur once every 3 months, but only if required. eg. If there were only patch releases in the last 3 months, no feature release will occur.
-    - The latest feature release will receive any security or bug fixes as patch releases
-    - Each feature release will have a separate git branch
-1. Patch releases
-    - Patch releases are used to fix bugs and security issues
-    - They are released as soon as they are ready
-1. LTS (Long-Term Support) Releases
-    - Every fourth release is an LTS release
-    - LTS releases are listed in the documentation
-    - LTS releases receive security patches
-    - LTS releases are supported for 5 feature releases, allowing for 15 months of support with a 3 month switchover time to the next LTS
+
 1. Development releases
-    - Development have the `-dev.N` suffix
-    - They are used to test new code before an official release is made
+    - Development versions have the `-dev.N` suffix
+    - They are used to test new code in QA
```

Comment:
Do we want a new dev release for each feature that goes to QA?

It might make sense once the pace of development slows down somewhat, but at the moment I expect it'll be more of a barrier than a help.

Reply:
Yeah I guess we don't need it, we can rely on just the SHA hashes. Should we get rid of dev releases completely then?

Reply:
Dev releases may still be useful, especially if we have some experimental features that we want to deploy somewhere less volatile than a shared QA environment. I think we can leave them in, but remove the item about using them for QA deployments.

Somewhat unrelated: Do we want a version check in version-tagged image builds to make sure the `pyproject.toml` version matches the tag?

Reply:
Yeah we can add that, good idea. You can move a tag to a different commit right?